api/v1beta1/cloudstackmachine_webhook_test.go

```diff
@@ -46,6 +46,16 @@ var _ = Describe("CloudStackMachine webhook", func() {
 			Expect(k8sClient.Create(ctx, dummies.CSMachine1)).Should(Succeed())
 		})
 
+		It("should accept a CloudStackMachine with disk Offering size attribute", func() {
+			dummies.CSMachine1.Spec.DiskOffering = dummies.DiskOffering3
+			Expect(k8sClient.Create(ctx, dummies.CSMachine1)).Should(Succeed())
+		})
+
+		It("should not accept a CloudStackMachine with disk Offering size attribute", func() {
+			dummies.CSMachine1.Spec.DiskOffering = dummies.DiskOffering4
```

Comment:
I'm guessing this test is about putting a bad attribute into the test, but DiskOffering4 isn't very descriptive either.

Another thing we could do, instead of having so many disk offering dummy vars is just set the single relevant attribute for the `It` block.

The dummy var attributes are reset for every `it` block.